### PR TITLE
Payments in #73

### PR DESCRIPTION
**Pull request by eric-silva-61**
### What this PR does (required):
- Collects user information and sends to Stripe
- Lists payment methods if available, otherwise shows add form

### Screenshots / Videos (required):
image
image

### Any information needed to test this feature (required):
- Currently you can only do this once per user since the setup intent that is generated on the server gets used.

### Any issues with the current functionality (optional):
- There are many features missing from this.  Updating payment method, deleting PM, editing etc

### CODE REVIEW

File: client/src/components/PaymentForm/PaymentForm.tsx [hunk 1]
```diff
@@ -39,45 +43,52 @@ const PaymentForm: FC = (): JSX.Element => {
...
-    //         state,
-    //         postal_code,
-    //       },
-    //     },
-    //   },
-    // });
-    // console.log('[PaymentMethod]', payload);
+    const response = await fetch('/payments/secret');
+    const { client_secret: clientSecret } = await response.json();
+    const result = await stripe.confirmCardSetup(clientSecret, {
```
Comment (rajivtitus) [inline client/src/components/PaymentForm/PaymentForm.tsx:63]: As discussed on our call today, I'm, not sure if we are creating a customer as I don't see the call to that specific API endpoint. 

Just adding this here for additional reference: https://stripe.com/docs/payments/save-and-reuse?platform=web

Comment (eric-silva-61) [reply client/src/components/PaymentForm/PaymentForm.tsx:63]: I think that we ended up with some overlap during dev.  Jeri is creating the user on stripe and prepopulating a setupintent in the User document.  I use her `const response = await fetch('/payments/secret');`  which then uses up that intent and returns the secret.  This does limit us to one payment method.  Her and I can probably talk about this next week and sort out the flow.

Comment (jerixmx) [reply client/src/components/PaymentForm/PaymentForm.tsx:63]: Hello, this is the part where the customer is created. I think I will have to make a separate controller for the customer creation so we could have multiple setupIntents. 
https://github.com/hatchways/team-old-fashioned/blob/9d51fb075d2a1ada9c71d10c86d49760ffcff3ed/server/controllers/payment.js#L6-L19

Comment (jerixmx) [reply client/src/components/PaymentForm/PaymentForm.tsx:63]: This will still have to be called on the front end though. Yup, let's discuss next week!

**Review by rajivtitus — APPROVED** [review]
Reviewed the new changes and it LGTM!

**Pull request merged**